cellprofiler/modules/measureobjectintensitydistribution.py

```diff
@@ -167,6 +164,10 @@ def __image_resource(filename):
 
 
 class MeasureObjectIntensityDistribution(cellprofiler.module.Module):
+    __doc__ = __doc__.format(**{
```

Comment:
When imports are needed to render documentation, the conventional module structure is:

```python
import foo
import bar
import baz

__doc__ = """
MyModule
========

{SOME_ELEMENT}\
""".format(**{
    "SOME_ELEMENT": foo.render(some_element)
})

class MyModule(cellprofiler.module.Module):
    # module stuff
```

Reply:
Gotcha, my concern was that the imports before the docstring and the `format` call on the docstring were causing python not to set `self.__doc__` properly. 
```
>>> class Foo(object):
...   """Here's my docstring {}""".format("test")
... 
>>> a = Foo()
>>> a.__doc__
>>> a.__doc__ is None
True
```

Reply:
TIL use `__doc__ = """...""".format(...)`:

```python
In [1]: class Foo(object):
   ...:     """Here is my docstring"""
   ...:

In [2]: class Bar(object):
   ...:     """Here is my {}""".format("docstring")
   ...:

In [3]: class Baz(object):
   ...:     __doc__ = """Here is my {}""".format("docstring")
   ...:

In [4]: foo = Foo()

In [5]: foo.__doc__
Out[5]: 'Here is my docstring'

In [6]: bar = Bar()

In [7]: bar.__doc__

In [8]: baz = Baz()

In [9]: baz.__doc__
Out[9]: 'Here is my docstring'
```